src/backend/marsha/core/tests/test_api_xapi_statement.py

```diff
@@ -47,11 +46,16 @@ def test_xapi_statement_with_no_lrs_configured(self):
 
         self.assertEqual(response.status_code, 501)
 
-    @override_settings(LRS_URL=r"http://lrs.com/data/xAPI")
-    @override_settings(DJANGO_LRS_AUTH_TOKEN=r"Basic ThisIsABasicAuth")
     def test_xapi_statement_api_with_invalid_payload(self):
         """Payload should follow a given pattern."""
-        video = VideoFactory()
+        video = VideoFactory(
+            playlist=PlaylistFactory(
+                consumer_site=ConsumerSiteFactory(
+                    lrs_url=r"http://lrs.com/data/xAPI",
+                    lrs_auth_token=r"Basic ThisIsABasicAuth",
+                )
+            )
+        )
```

Comment:
You don't need to call on 3 factories. This should do the same:
```python
video = VideoFactory(
    playlist__consumer_site__lrs_url="http://lrs.com/data/xAPI",
    playlist__consumer_site__lrs_auth_token="Basic ThisIsABasicAuth",
)
```

Curious, why did you put raw strings?

Reply:
> why did you put raw strings?

No idea :laughing:

Reply:
:ok_hand: